src/structures/CommandInteractionOptionResolver.js

```diff
@@ -216,6 +216,17 @@ class CommandInteractionOptionResolver {
     return option?.role ?? null;
   }
 
+  /**
+   * Gets an attachment option.
+   * @param {string} name The name of the option.
+   * @param {boolean} [required=false] Whether to throw an error if the option is not found.
+   * @returns {?(MessageAttachment|APIAttachment)} The value of the option, or null if not set and not required.
```

Comment:
This doesn't return the value of the option, it returns the attachment. Isn't this description just redundant?

Reply:
I just copied the same description format from the other option types

Reply:
That doesn't change the fact that this doesn't return the value of the attachment, it returns the attachment itself

Reply:
@amitojsingh366 you missed this

Reply:
oh my bad

Reply:
im gonna keep this as is cuz kyra said it's fine like this